roles/system/tasks/main.yml

```diff
@@ -17,14 +17,14 @@
   when: ansible_lsb.id == "Debian"
   tags: ['master','update']
 
-- name: Run apt-get update  
-  shell: apt-get update ; echo
-  register: result
+- name: Update APT cache, not more than once every 8 hours
+  apt: 
+    update_cache: yes 
+    cache_valid_time: 28800
   tags: ['master','custom','update']
 
 - name: Fix broken package
   shell: dpkg --configure -a
-  when: "'dpkg --configure -a' in result.stderr"
```

Comment:
Yes, this is a choice to use apt module. I didn't look at apt module output to see if behavior is the same as previous apt-cache update command. Even if this is highly probable.
I decided the new behavior (running the task each time) is not really an issue as the command is very fast when there's nothing to do.